src/components/paragraph/sul-home-banner/sul-home-banner.client.tsx

```diff
@@ -52,11 +52,15 @@ export const SulHomeBannerFormClient = () => {
         </label>
         <select
           id={`${inputId}-action`}
-          className="h-40 w-full border-0 bg-none text-16 font-semibold hover:cursor-pointer md:w-auto md:min-w-[15rem] md:text-20 xl:text-22"
+          className="h-40 w-full border-0 bg-none text-16 font-semibold leading-normal hover:cursor-pointer md:w-auto md:min-w-[15rem] md:text-20 xl:text-22"
           onChange={e => setFormAction(e.target.value)}
           value={formAction}
         >
-          <option value="/all">All resources</option>
+          <option value="/all">All library resources</option>
+          <option value="/catalog">Catalog</option>
```

Comment:
@pookmish Darcey would like to add Catalog and Articles as search options.

I cannot seem to figure out what I'm missing to get this to work. Are there additional permissions needed before we can redirect to searchworks.stanford.edu? Do I need to add a redirect on the Drupal side of things? 🤔 
![image](https://github.com/user-attachments/assets/f4ac47bc-604c-4b15-9c1e-cb235d4a1c78)

Reply:
just set the it to `value="https://searchworks.stanford.edu/"`. There's no need to do any redirecting. the form `action` value will change and when the form is submitted, the browser will just go straight to it. the `/catalog` page doesn't exist.

Reply:
Alternatively, set the value to `https://library.stanford.edu/all#catalog` if they want to stay on this domain.